src/platform-includes/replay/privacy-configuration/javascript.mdx

```diff
@@ -1,13 +1,13 @@
 The following options can be configured as options to the integration, in `new Replay({})`:
 
-| key           | type                     | default                                  | description                                                                                                                                                                                       |
-| ------------- | ------------------------ | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| mask          | string[]                 | `.sentry-mask, [data-sentry-mask]`       | Mask all elements that match the given DOM selectors. See [Masking](#masking) section for an example. Note that any configured selectors will be in _addition_ to the defaults.                   |
-| maskAllText   | boolean                  | `true`                                   | Mask _all_ text content. Will pass text content through `maskFn` before sending to server.                                                                                                        |
-| maskAllInputs | boolean                  | `true`                                   | Mask values of `<input>` elements. Passes input values through `maskFn` before sending to server.                                                                                                 |
-| block         | string[]                 | `.sentry-block, [data-sentry-block]`     | Redact all elements that match the DOM selector(s). See [Blocking](#blocking) section for an example. Note that any configured selectors will be in _addition_ to the defaults.                   |
-| blockAllMedia | boolean                  | `true`                                   | Block _all_ media elements (`img`, `svg`, `video`, `object`, `picture`, `embed`, `map`, `audio`).                                                                                                 |
-| ignore        | string[]                 | `.sentry-ignore, [data-sentry-ignore]`   | Ignores all events on the matching input fields. See [Ignoring](#ignoring) above for an example.                                                                                                  |
-| maskFn        | (text: string) => string | `(text) => '*'.repeat(text.length)`      | Function to customize how text content is masked before sending to server. By default, masks text with `*`.                                                                                       |
```

Comment:
Why remove the `: string` here?

Reply:
If you ask me we should prioritize correctness over looks here 😄 But I'll leave it up to you.